src/labeled_signal.jl

```diff
@@ -135,15 +135,22 @@ function get_labels(labels::SignalV2, span_relative_to_recording)
 end
 
 """
-    label_signals(signals, annotations; groups=:recording, kwargs...)
+    label_signals(signals, annotations;
+                    groups=:recording,
+                    labels_column,
+                    epoch,
+                    encoding,
+                    roundto=nothing)
 
 Create a "labeled signals" table from a signals table and a table of annotations
 containing labels.
 
-Annotations will be passed to [`labels_to_samples_table`](@ref) and additional
-kwargs are forwarded there as well.  Default values there are
-- `labels_column` the column in the annotations table with labels,
-- `epoch` the sampling period of the labels
+Keyword arguments:
+- `groups`: the column to group over, defaults to `:recording`.
+- `labels_column`: the column in the annotations table containing the labels.
+- `epoch`: the sampling period of the labels.
+- `encoding::Dict`: the label -> `UInt8` mapping to use for encoding the labels.
+- `roundto`: controls rounding of "shaggy spans", defaults to `nothing` for no rounding.
```

Comment:
```suggestion
Annotations will be passed to [`labels_to_samples_table`](@ref), as well as 
kwargs.  `labels_to_samples_table` requires these keyword arguments:
- `groups`: the column to group over, defaults to `:recording`.
- `labels_column`: the column in the annotations table containing the labels.
- `epoch`: the sampling period of the labels.
- `encoding::Dict`: the label -> `UInt8` mapping to use for encoding the labels.
- `roundto`: controls rounding of "shaggy spans", defaults to `nothing` for no rounding.
```

let's still call out this link, as well as listing out all teh required kwargs